libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/kotlin/org/jetbrains/kotlin/gradle/Kapt3IT.kt

```diff
@@ -67,6 +67,32 @@ class Kapt3WorkersIT : Kapt3IT() {
             assertSuccessful()
         }
     }
+
+    private fun testSimpleWithCustomJdk(gradleVersion: String, javaHome: File) {
+        val gradleVersionRequired = GradleVersionRequired.AtLeast(gradleVersion)
+
+        // Skip test if the specified jdk isn't available in the test environment.
+        if (!javaHome.isDirectory)
+            return
```

Comment:
I recommend to replace `return` with `org.junit.Assume#assumeTrue`.
When assumption is violated, a test is also skipped (it is not considered failed),
but it is shown as ignored in IJ and in TeamCity test reports.